### DISCUSSION TITLE
config.php not configured?

### DISCUSSION THREAD

**Discussion by bjchapm** [nextcloud/all-in-one#1357] [Questions] [answered]
Hello,

I've just installed AIO using the docker-compose.yml file with Caddy enabled as a reverse proxy. After installation, I'm taken to a Nextcloud install screen as though there is no configuration. Perhaps the proper config.php file is not being copied into place during the AIO installation?

![Screen Shot 2022-11-03 at 11 33 22 AM](https://user-images.githubusercontent.com/6674822/199764952-4557b13d-d10d-4364-aa77-4d6122e5921c.png)

How can I troubleshoot this? 

Thank you for any pointers, Ben

**Comment by szaimen** [comment]
hm... can you post the logs of the `nextcloud-aio-nextcloud` container here?

**Reply by bjchapm** [reply]
Thanks for your response! Please see below - advise if more is needed:

nc: getaddrinfo: Name does not resolve
Waiting for database to start...
              now              
-------------------------------
 2022-11-03 11:08:21.829781-04
(1 row)

Configuring Redis as session handler...
Setting php max children...
Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Applying one-click-instance settings...
Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                                
 Command "app:enable" is not defined.                                           
                                                                                

 Do you want to run "app:check-code" instead?  (yes/no) [no]:
 > Adjusting log files...
Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                
  There are no commands defined in the "config:app" namespace.  
                                                                

Applying network settings...
Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                        
  There are no commands defined in the "maintenance:update" namespace.  
                                                                        
  Did you mean this?                                                    
      maintenance                                                       
                                                                        

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                
  There are no commands defined in the "config:app" namespace.  
                                                                

Nextcloud is not installed - only a limited number of commands are available

                                                                                
 Command "app:enable" is not defined.                                           
                                                                                

 Do you want to run "app:check-code" instead?  (yes/no) [no]:
Nextcloud is not installed - only a limited number of commands are available

                                         
  Command "app:install" is not defined.  
                                         
  Did you mean one of these?             
      app:check-code                     
      maintenance:install                
                                         

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                
  There are no commands defined in the "config:app" namespace.  
                                                                

Nextcloud is not installed - only a limited number of commands are available

                                         
  Command "app:install" is not defined.  
                                         
  Did you mean one of these?             
      app:check-code                     
      maintenance:install                
                                         

Nextcloud is not installed - only a limited number of commands are available

                                                                
  There are no commands defined in the "config:app" namespace.  
                                                                

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "config:system" namespace.  
                                                                   

 > waiting for Fulltextsearch to become available...
waiting for Fulltextsearch to become available...
waiting for Fulltextsearch to become available...
Nextcloud is not installed - only a limited number of commands are available

                                         
  Command "app:install" is not defined.  
                                         
  Did you mean one of these?             
      app:check-code                     
      maintenance:install                
                                         

Nextcloud is not installed - only a limited number of commands are available

                                         
  Command "app:install" is not defined.  
                                         
  Did you mean one of these?             
      app:check-code                     
      maintenance:install                
                                         

Nextcloud is not installed - only a limited number of commands are available

                                         
  Command "app:install" is not defined.  
                                         
  Did you mean one of these?             
      app:check-code                     
      maintenance:install                
                                         

Nextcloud is not installed - only a limited number of commands are available

                                                                    
  There are no commands defined in the "fulltextsearch" namespace.  
                                                                    

Nextcloud is not installed - only a limited number of commands are available

                                                                               
  There are no commands defined in the "fulltextsearch_elasticsearch" namespa  
  ce.                                                                          
                                                                               

Nextcloud is not installed - only a limited number of commands are available

                                                                          
  There are no commands defined in the "files_fulltextsearch" namespace.  
                                                                          

Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Activating collabora config...
Nextcloud is not installed - only a limited number of commands are available

                                                                   
  There are no commands defined in the "richdocuments" namespace.  
                                                                   

Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}
Exception: Not installed in /var/www/html/lib/base.php:282

ace:
#0 /var/www/html/lib/base.php(670): OC::checkInstalled(Object(OC\SystemConfig))
#1 /var/www/html/lib/base.php(1111): OC::init()
#2 /var/www/html/cron.php(43): require_once('/var/www/html/l...')

}

**Comment by szaimen** [comment]
Okay. This looks like a hickup. Please try again by firsr properly cleaning your server https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

**Reply by bjchapm** [reply]
Will do - thanks again - I will reinstall and report back

**Comment by szaimen** [comment]
One question: is your disk space full maybe?

**Comment by szaimen** [comment]
Did it work or did you run into the same issue again?

**Comment by bjchapm** [comment, marked as answer]
I apologize for the delay in responding! It failed again, but I realized that it was an error that I made. I failed to clear the old data from the volume that contained the previous installation, which I think prevented the installer from working correctly. Everything is functioning correctly now, after a re-installation. 

Thank you for the monumental amount of work that you have done on this project.

Best wishes, Ben

**Reply by szaimen** [reply]
All right! Great, that you found the issue and solved it!
